concent_api/core/tests/test_integration_accept_or_reject_results.py

```diff
@@ -1117,7 +1117,7 @@ def test_requstor_send_again_subtask_results_accepted_or_rejected_when_message_a
 
         self._test_400_response(response_2)
 
-    def test_requstor_send_again_subtask_results_accepted_or_rejected_when_message_already_rejected_concent_should_return_http_400(self):
+    def test_requestor_send_again_subtask_results_accepted_or_rejected_when_message_already_rejected_concent_should_return_http_400(self):
```

Comment:
Maybe typo fixes should be separated commit? I think @cameel would point that.

Reply:
:+1: